src/core/constants.js

```diff
@@ -22,8 +22,18 @@ export const VERSION =
  */
 export const P2D = 'p2d';
 /**
- * One of the two render modes in p5.js: P2D (default renderer) and WEBGL
- * Enables 3D render by introducing the third dimension: Z
+ * One of the two render modes in p5.js, most known for 3D rendering.
+ *
+ * Some differences and new features to expect compared to the default render mode <a href="/#/p5/P2D">P2D</a>:
+ * - The <strong>coordinate system</strong> is centered and uses an optional z-dimension
```

Comment:
Also echoing what Austin said. I think for this list it would be good to put the topic as the first word, and then follow it with a sentence or two about how it is different. For example:

**Coordinate System** - When drawing in WEBGL mode, the origin point (0,0,0) is located at the center of the screen. More information about WEBGL [coordinates and transformations can be found on the learn page](https://p5js.org/learn/getting-started-in-webgl-coords-and-transform.html)

**Next topic** - one or two sentences describing the differences with a link to more info if possible.